pkg/wave/buffer_test.go

```diff
@@ -12,8 +12,9 @@ func TestBufferStoreCopyAndLoad(t *testing.T) {
 		SamplingRate: 48000,
 	}
 	testCases := map[string]struct {
-		New    func() EditableAudio
-		Update func(EditableAudio)
+		New      func() EditableAudio
+		Update   func(EditableAudio)
+		Validate func(Audio, Audio) bool
```

Comment:
```suggestion
		Validate func(*testing.T, Audio, Audio) bool
```
would be better to make it easy to find why the validation is failed.

Reply:
I see. In that case, I'll just return error then instead, and let the higher level to print that.

Reply:
`t.Error` outputs the filename and line number where it errored. Calling `t.Error` in Validate func makes it easy to find the failing line.

Reply:
Oh! I didn't know that. I'll do that then!

Reply:
Updated.